doc/ansible/user-guide.md

```diff
@@ -395,6 +395,26 @@ kubectl logs deployment/memcached-operator -c operator
 The `ansible` logs contain all of the information about the Ansible run and will make it much easier to debug issues within your Ansible tasks,
 whereas the `operator` logs will contain much more detailed information about the Ansible Operator's internals and interface with Kubernetes.
 
+### Additional Ansible debug
+
+Occasionally while developing additional debug in the Operator logs is nice to have. To enable Ansible debug output, ie `-vvvv`.
+Add the following to the Operator manifest.
```

Comment:
can we maybe add `operator.yaml` to this line? It took me a second to figure out what we were referring to.